ballerina/types.bal

```diff
@@ -263,3 +257,39 @@ public type MQCIH record {|
     string nextTransactionId = "";
     int inputItem = 0;
 |};
+
+# Header record representing the MQIIH structure.
+#
+# + flags - Flag of the header  
+# + encoding - Numeric encoding of data that follows NameValueString  
+# + strucId - Structure identifier  
+# + strucLength - Length of the structure  
+# + version - Structure version number  
+# + codedCharSetId - Character set identifier of data that follows NameValueString  
+# + format - Format name of data that follows NameValueString  
+# + lTermOverride - The logical terminal override, placed in the IO PCB field
+# + mfsMapName - The message format services map name, placed in the IO PCB field
+# + replyToFormat - This is the MQ format name of the reply message that is sent 
+#                   in response to the current message
+# + authenticator - The RACF® password or PassTicket
+# + tranInstanceId - This is the transaction instance identifier
+# + tranState - This indicates the IMS conversation state
+# + commitMode - IMS commit mode
+# + securityScope - This indicates the IMS security processing required
+public type MQIIH record {|
+    int flags = 0;
+    int encoding = 0;
+    string strucId = "IIH ";
+    int strucLength = 84;
+    int version = 1;
+    int codedCharSetId = 0;
+    string format = "        ";
+    string lTermOverride = "        ";
+    string mfsMapName = "        ";
+    string replyToFormat = "        ";
+    string authenticator = "        ";
```

Comment:
Can we define a common constant for the default value and reuse in here? 

```suggestion
    string format = "        ";
    string lTermOverride = "        ";
    string mfsMapName = "        ";
    string replyToFormat = "        ";
    string authenticator = "        ";
```

Reply:
Fixed